sdk/communication/test-resources/test-resources.json

```diff
@@ -82,15 +82,15 @@
     },
     "COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING": {
       "type": "string",
-      "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2021-10-01-preview').primaryConnectionString]"
+      "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2023-03-31').primaryConnectionString]"
```

Comment:
the changes in 85 & 93th lines are not applied for other platforms, should we apply for others?

Reply:
I had already done that and they are merged to main